.github/workflows/test.yml

```diff
@@ -209,7 +209,7 @@ jobs:
           esac
 
       - name: "Save HF models cache"
-        if: ( steps.changed-src-files.outputs.any_changed == 'true' && github.event_name != 'pull_request' && strategy.job-index == 0 )
+        if: ( steps.changed-src-files.outputs.any_changed == 'true' && github.event_name != 'pull_request' )
```

Comment:
If we save the cache for all jobs, there can be concurrent cache write errors. I had added the `strategy.job-index == 0` condition to make sure it only got saved once per workflow run.

Reply:
@ckadner are those concurrent write errors only for concurrent writes with the same cache key? In this case we're trying to write multiple smaller cache entries